xenit-alfresco/values.yaml

```diff
@@ -25,6 +25,8 @@ ingress:
   ingressAnnotations:
     kubernetes.io/ingress.class: "nginx"
     cert-manager.io/cluster-issuer: "letsencrypt-production"
+  ingressClass:
+    enabled: true
```

Comment:
Shouldn't this be more like `ingressClass: "nginx"`? So you can configure the ingress class instead of hardcoding it to nginx

Reply:
Can be done. I hardcoded it to nginx since that is the default for our chart?